guides/managing-sessions.rst

```diff
@@ -43,7 +43,7 @@ one single container object:
 
 .. note::
 
-    Mink will even lazy-start your sessions when needed (on first ``getSession()``
+    Mink will even lazy-start your sessions when needed (on the first ``getSession()``
```

Comment:
actually, this doc is outdated. The Mink class does not ever start session anymore. It is a simple registry. Sessions themselves are now starting automatically when you visit a page for the first time.